PicApp_git/PicApp/app/Http/Controllers/FollowsController.php

```diff
@@ -19,4 +20,15 @@ public function destroy($id)
         Auth::user()->unfollow($id);
         return back();
     }
+
+    public function following_user_index(Request $request, $id)
+    {
+        $user = User::find($id);
+        $followings = $user->followings()->get();
+        $data = [
+            'user' => $user,
+            'users' => $followings,
```

Comment:
$usersだと若干$userと混同しそうなので、より明確な変数名にしておきましょう
```suggestion
            'following_users' => $followings,
```